parser.y

```diff
@@ -3979,6 +3982,8 @@ ExplainStmt:
 			Stmt:	$5,
 			Format: $4,
 		}
+		startOffset := parser.startOffset(&yyS[yypt])
+		$5.SetText(string(parser.src[startOffset:]))
 	}
 |	ExplainSym "FORMAT" "=" ExplainFormatType "FOR" "CONNECTION" NUM
```

Comment:
We cannot get the optimizer trace from `explain for connection` since the plan may have already been generated?